### ISSUE TITLE
xi-tui only works with a log_config.yaml file in the working directory

### ISSUE THREAD

**Issue by mssun** [little-dude/xi-tui#43]
xi-tui will first open log_config.yaml when opening:

https://github.com/little-dude/xi-tui/blob/master/src/main.rs#L62

If there is no log_config.yaml, unwrap will fail.

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Log4rs(Error { repr: Os { code: 2, message: "No such file or directory" } })', /checkout/src/libcore/result.rs:906:4
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
             at /checkout/src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at /checkout/src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at /checkout/src/libstd/sys_common/backtrace.rs:60
             at /checkout/src/libstd/panicking.rs:381
   3: std::panicking::default_hook
             at /checkout/src/libstd/panicking.rs:397
   4: std::panicking::rust_panic_with_hook
             at /checkout/src/libstd/panicking.rs:577
   5: std::panicking::begin_panic
             at /checkout/src/libstd/panicking.rs:538
   6: std::panicking::begin_panic_fmt
             at /checkout/src/libstd/panicking.rs:522
   7: rust_begin_unwind
             at /checkout/src/libstd/panicking.rs:498
   8: core::panicking::panic_fmt
             at /checkout/src/libcore/panicking.rs:71
   9: core::result::unwrap_failed
             at /checkout/src/libcore/macros.rs:23
  10: <core::result::Result<T, E>>::unwrap
             at /checkout/src/libcore/result.rs:772
  11: xi_tui::run
             at src/main.rs:62
  12: xi_tui::main
             at src/main.rs:44
  13: __rust_maybe_catch_panic
             at /checkout/src/libpanic_unwind/lib.rs:99
  14: std::rt::lang_start
             at /checkout/src/libstd/panicking.rs:459
             at /checkout/src/libstd/panic.rs:361
             at /checkout/src/libstd/rt.rs:59
  15: main
  16: __libc_start_main
             at ../csu/libc-start.c:308
  17: _start
```

**Comment by little-dude** [comment]
Yeah I'm getting rid of that soon, this is bad. I was just convenient in early stages to get something working without adding code for logging.

**Comment by little-dude** [comment]
This should be fixed by d6da168e20d1709e3be4bfdf96eb4bcc19bef753

**Comment by mssun** [comment]
Great, thanks @little-dude .
